bitnami/thanos/values-production.yaml

```diff
@@ -175,6 +175,9 @@ querier:
     ## Service type
     ##
     type: ClusterIP
+    ## Thanos Querier service clusterIP IP
+    ##
+    ClusterIP: ""
```

Comment:
Instead of setting it to `""` by default, please use the approach below:
```yaml
  ## Set the Cluster IP to use
  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#choosing-your-own-ip-address
  ##
  # clusterIP: None
```

> Note: use camelCase style: `ClusterIP` -> `clusterIP`

Please apply this suggestion to the rest of parameter you added.

Reply:
Done